src/properties.jl

```diff
@@ -116,3 +118,37 @@ filter(φ::SphericalVector, b::Array) = ( length(b) <= 1 ? true :
          ( abs(sum(bi.m for bi in b)) <= getL(φ) )  ) )
 
 rot3Dcoeffs(::SphericalVector, T::DataType=Float64) = Rot3DCoeffs(T)
+
+
+struct SphericalMatrix{L1, L2, LEN1, LEN2, T} <: AbstractProperty
+   val::SMatrix{LEN1, LEN2, T}
+   _valL1::Val{L1}
+   _valL2::Val{L2}
+end
+
+getL(φ::SphericalMatrix{L1,L2}) where {L1,L2} = [L1;L2]
```

Comment:
please return a tuple here: `... where {L1, L2} = L1, L2`

* `[L1; L2]` is actually a matrix; to construct a vector use `[L1, L2]`. 
* Such a vector is allocating, whereas a tuple is on the stack. Further, the values `L1, L2` can be inferred if they are returned as a tuple.